include/GafferUI/ViewportGadget.h

```diff
@@ -125,6 +125,11 @@ class GAFFERUI_API ViewportGadget : public Gadget
 		void setDragTracking( bool dragTracking );
 		bool getDragTracking() const;
 
+		/// When variable aspect zoom is enabled, the two axis can be scaled
+		/// independently when performing a 2D zoom.
+		void setVariableAspectZoom( bool variableAspectZoom );
```

Comment:
@danieldresser, any comments as to how this will interact with the work you're doing?